lte/gateway/c/core/oai/tasks/amf/amf_recv.cpp

```diff
@@ -79,6 +81,18 @@ int amf_handle_service_request(
         " receved TMSI %08X stored TMSI %08X \n",
         ue_id, tmsi_rcv, tmsi_stored);
 
+    paging_ctx = &ue_context->paging_context;
+
+    if ((paging_ctx) &&
+        (NAS5G_TIMER_INACTIVE_ID != paging_ctx->m5_paging_response_timer.id) &&
+        (0 != paging_ctx->m5_paging_response_timer.id)) {
+      amf_app_stop_timer(paging_ctx->m5_paging_response_timer.id);
+      paging_ctx->m5_paging_response_timer.id = NAS5G_TIMER_INACTIVE_ID;
+      paging_ctx->paging_retx_count           = 0;
+      // Fill the itti msg based on context info produced in amf core
+      OAILOG_INFO(LOG_AMF_APP, "T3513: After stopping PAGING Timer\n");
+    }
```

Comment:
We may in the magma [doc](https://docs.magmacore.org/docs/contributing/contribute_conventions) describe better what falls in each log level. Here I would suggest at least DEBUG loglevel.

Reply:
Done the changes as suggested